tests/framework/evaluators/gist_agent_evaluator.py

```diff
@@ -35,10 +35,23 @@ def test_one(self, verify_value: Any, test_value: Any):
         pass_fail: bool = self.ask_llm(acceptance_criteria=str(verify_value),
                                        text_sample=str(test_value))
 
+        did_str: str = "didn't"
         if self.negate:
-            self.asserts.assertFalse(pass_fail)
+            did_str = "did"
+
+        message: str = f"""
+text_sample unexpectedly {did_str} match acceptance criteria.
+
+text_sample:
+{test_value}
+
+acceptance_criteria:
+{verify_value}
+"""
```

Comment:
Add extra information to the GistAgentEvaluator so there is more context as to how the test failed other than an uninformative False != True